REPO_LAYOUT.md

```diff
@@ -9,7 +9,7 @@ are:
 * [bazel/](bazel/): Configuration for Envoy's use of [Bazel](https://bazel.build/).
 * [ci/](ci/): Scripts used both during CI as well as to build Docker containers.
 * [configs/](configs/): Example Envoy configurations.
-* [docs/](docs/): End user facing Envoy proxy and data plane API documentation as well as scripts
+* [docs/](docs/): End users facing Envoy proxy and data plane API documentation as well as scripts
```

Comment:
I think the former usage is how I would generally see this phrase written and spoken, so I'd prefer not to change this one. But this phrase is a bit awkward overall ("end user" makes me think of the HTTP request initiator in a front proxy context, which is not what was intended).

What do you think about calling this "Envoy's installation, operation, and data plane API documentation as well as scripts for publishing final docs during releases." ?

Reply:
I think your suggestion is better than currrent one !